### ISSUE TITLE
Change the sort order of entries

### ISSUE THREAD

**Issue by Dev-iL** [pragmatic-streamlit/streamlit-file-browser#33]
I'm looking for a way to sort the files in order of decreasing modified date (newest first). This functionality seems absent from the GUI (i.e. clicking the title doesn't do anything), but is it possible to specify some flag somewhere so that the static list is sorted as necessary?

**Comment by y805939188** [comment]
We had released a new version 3.2.20. You can use it like:
```python
def sort(files):
        return sorted(files, key=lambda x: x["size"])
    
event = st_file_browser(your_path, sort=sort)
```

**Comment by y805939188** [comment]
> I'm looking for a way to sort the files in order of decreasing modified date (newest first). This functionality seems absent from the GUI (i.e. clicking the title doesn't do anything), but is it possible to specify some flag somewhere so that the static list is sorted as necessary?

We had released a new version 3.2.20. You can use it like:
```python
def sort(files):
        return sorted(files, key=lambda x: x["size"])
    
event = st_file_browser(your_path, sort=sort)
```
I hope this can be helpful to you.

**Comment by Dev-iL** [comment]
Thanks! I will give it a try.
